@bshyong @ncksllvn 
I was able to re-create this issue on my feature branch `10684-formation-breadcrumb` by doing the following:

1. `$ yarn remove core-js`
2. `$ yarn install`
3. `$yarn watch`

I received the following error:

```shell
ERROR in ./src/platform/polyfills/index.js
Module not found: Error: Can't resolve 'core-js/modules/es7.string.pad-end' in '/Users/<USERNAME>/Development/vets-website/src/platform/polyfills'
 @ ./src/platform/polyfills/index.js 86:0-44
 @ multi ./src/platform/polyfills history react react-dom react-redux react-router redux redux-thunk raven-js
```

I'm not sure this is directly related to Formation, but adding `core-js` to the devDependencies does allow the `$ yarn watch` command to build Metalsmith and Webpack as expected.

--- a/package.json
+++ b/package.json
@@ -93,6 +93,7 @@
     "classlist-polyfill": "^1.0.3",
     "classnames": "^2.2.5",
     "command-line-args": "^3.0.1",
+    "core-js": "^2.5.7",
     "cors": "^2.7.1",
     "css-loader": "^0.28.10",
     "dataset": "^0.3.1",

